Source/Controllers/ESTabBarController.m

```diff
@@ -268,7 +307,16 @@ - (void)customizeButtons {
         [button customizeForTabBarWithImage:self.tabIcons[i]
                               selectedColor:self.selectedColor ?: [UIColor blackColor]
                                 highlighted:isHighlighted];
-
+        
+        UILabel *badgeLabel = self.badgeLabels[i];
```

Comment:
Same comment for this, would be nice if we can take this logic to a `- (void)customizeLabel` method.